domain/domain.go

```diff
@@ -282,6 +282,7 @@ func (do *Domain) Reload() error {
 	defer do.m.Unlock()
 
 	var err error
+	var latestSchemaVersion int64
```

Comment:
It doesn't reach three or more variables. I think there is no need to change this format.